presto-docs/src/main/sphinx/release/release-0.179-t.rst

```diff
@@ -4,13 +4,139 @@ Release 0.179-t
 
 Presto 0.179-t is equivalent to Presto release 0.179, with some additional features and patches.
 
-**Spill to disk support for joins**
+Cost-Based Optimizer
+--------------------
+* Support cost-based join reordering. See :doc:`../optimizer/reorder-joins`.
+* Replace the ``distributed_joins`` property with the ``join_distribution_type`` session property or ``join-distribution-type`` config property.  Options are ``AUTOMATIC``, ``REPARTITIONED``, and ``REPLICATED``.
+* Replace the ``reorder_joins`` property with the ``join_reordering_strategy`` session property or ``optimizer.join-reordering-strategy`` config property.  Options are ``NONE``, ``ELIMINATE_CROSS_JOINS``, and ``COST_BASED``.
+* Determine join distribution type based on statistics when ``join_distribution_type`` is set to ``AUTOMATIC``.
+* Statistics calculated by Presto for each stage of the query plan can be seen in ``EXPLAIN`` results.
```

Comment:
Show estimated statistics for each stage of the query plan in ``EXPLAIN`` results.